pyfar/constants/air_attenuation.py

```diff
@@ -0,0 +1,243 @@
+"""Air attenuation calculation."""
+import numpy as np
+
+
+def air_attenuation_iso(
```

Comment:
Is there another air attenuation that we want to implement? Not sure if we need the ISO in the name. In other places, we also don't mention the source in the function name.